doc/CMakeLists.txt

```diff
@@ -8,7 +8,6 @@ set(MANPAGES
     saunafs.7
     saunafs-admin.8
     saunafs-cgiserver.8
-    saunafs-probe.8          # not a source
```

Comment:
Instead of removing the man file entirely, we should perhaps make it's own man file and clearly explain that it is deprecated and not supported.

Reply:
Reverted.